services/api/src/models/user.js

```diff
@@ -60,11 +58,4 @@ schema.pre('save', async function preSave(next) {
   return next();
 });
 
-schema.methods.toResource = function toResource() {
-  return {
-    id: this._id,
-    ...omit(this.toObject(), ['_id', '__v', 'hashedPassword', '_password'])
```

Comment:
Note that the previous code here was explicitly ensuring no `this._password` was being leaked. It may not be important because it's just a temporary field. But I would be more comfortable adding some unit tests around that case

Reply:
Cool just added some unit tests around `_password` and `hashedPassword` on users specifically.